app-center-services/src/main/java/org/exoplatform/appcenter/service/ApplicationCenterService.java

```diff
@@ -815,4 +860,15 @@ private Long getDefaultImageId() {
     return userApplicationsList;
   }
 
+  private  Application fromApplication(Application application) throws FileStorageException {
+    Application storedApplication = appCenterStorage.getApplicationById(application.getId());
```

Comment:
Is it needed to do a call to the storage on each application search result ? could we put all those field in ES ?